feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/governance/adapter/PushGovernanceRVItem.kt

```diff
@@ -0,0 +1,24 @@
+package io.novafoundation.nova.feature_push_notifications.data.presentation.governance.adapter
+
+import io.novafoundation.nova.feature_push_notifications.data.presentation.governance.PushGovernanceModel
+
+data class PushGovernanceRVItem(
+    val model: PushGovernanceModel,
+    val tracksText: String
+) {
+    val chainId = model.chainId
+
+    val governance = model.governance
+
+    val chainName = model.chainName
+
+    val chainIconUrl = model.chainIconUrl
+
+    val isEnabled = model.isEnabled
+
+    val isNewReferendaEnabled = model.isNewReferendaEnabled
+
+    val isReferendaUpdatesEnabled = model.isReferendaUpdatesEnabled
+
+    val isDelegationVotesEnabled = model.isDelegationVotesEnabled
```

Comment:
I suggest to make a simple data model and a mapper for consistensy with other parts of application

Reply:
I think it's consistent enough PushGovernanceModel is a part of presentation layer and I use `model` in `rvItem` before.
Also I think it's not a problem since `rvItem` and `model` both are declared in the directory